setup.py

```diff
@@ -279,17 +279,14 @@ def initialize_options(self):
 
         for root, dirs, files in os.walk('pandas'):
             for f in files:
-                if f in self._clean_exclude:
-                    continue
-
-                # XXX
-                if 'ujson' in f:
```

Comment:
`f` is this context is the filename (only). So the `ujson` conditional doesn't do anything since those filenames are already part of the list.

I thought about that but didn't want to break the way it worked before. So you are saying that all excluded files should be referred to by their project root relative path? e.g. `pandas/src/ujson/lib/ultrajsondec.c` I believe directories should be allowed for exclusion also.